cmake/VexCLBoostTargets.cmake

```diff
@@ -0,0 +1,46 @@
+
+# Support for CMake < 3.6 or so
+# Unlike the real targets in CMake 3.6+, this does not resolve all
+# dependencies based on Boost version. Use with care.
+# This also may be used when CMake < 3.11 (3.10?) and CMake is older than Boost
+
+if(NOT TARGET Boost::system)
+    add_library(Boost::system IMPORTED INTERFACE)
+    set_target_properties(Boost::system PROPERTIES
+        INTERFACE_LINK_LIBRARIES "${Boost_SYSTEM_LIBRARY}"
+        INTERFACE_INCLUDE_DIRECTORIES "${Boost_INCLUDE_DIRS}"
+        )
```

Comment:
I would say `set_target_properties` should go after `endif` here. (re https://github.com/ddemidov/vexcl/pull/245#issuecomment-389860544)